dataland-batch-manager/src/main/resources/application.properties

```diff
@@ -2,5 +2,7 @@ dataland.dataland-batch-manager.client-id=dataland-batch-manager
 dataland.dataland-batch-manager.client-secret=${DATALAND_BATCH_MANAGER_CLIENT_SECRET}
 dataland.dataland-batch-managet.get-all-gleif-companies.flag-file=${GET_ALL_GLEIF_COMPANIES_FLAG_FILE}
 dataland.backend.base-url=${INTERNAL_BACKEND_URL}
+dataland.dataland-batch-manager.mapping-file=/deltafiles/isinMapping.csv
 dataland.keycloak.base-url=http://keycloak:8080/keycloak
 gleif.download.baseurl=https://goldencopy.gleif.org/api/v2/golden-copies/publishes/lei2
+gleif.mapping.download.baseurl=https://mapping.gleif.org/api/v2/isin-lei
```

Comment:
1.) Since this is (in contrast to `gleif.download.baseurl`) already the complete url it should not say `baseurl` but rather `url` only.

2.) Maybe `isinmapping` instead of `mapping`?

Reply:
done